front-end/src/common/stats-generators/game-rating/RoleRatingStatGenerator.ts

```diff
@@ -35,4 +35,10 @@ export abstract class RoleRatingStatGenerator extends GameRatingStatGenerator {
   abstract getJunglerStatValue(playerStatsModel: AggregatedPlayerStatModel): number;
 
   abstract getSupportStatValue(playerStatsModel: AggregatedPlayerStatModel): number;
+
+  // abstract getSoloLaneStatValueGameModel(playerGameModels: PlayerGameModel[]): number;
+
+  // abstract getJunglerStatValueGameModel(playerGameModels: PlayerGameModel[]): number;
+
+  // abstract getSupportStatValueGameModel(playerGameModels: PlayerGameModel[]): number;
```

Comment:
Commented out code, if we dont need it lets remove

Reply:
Fixed